test/s1/appframework/appframework_test.go

```diff
@@ -120,32 +113,13 @@ var _ = Describe("s1appfw test", func() {
 
 			// Upload V1 apps to S3 for Standalone
 			testenvInstance.Log.Info(fmt.Sprintf("Upload %s apps to S3 for Standalone", appVersion))
-			s3TestDir = "s1appfw-" + testenv.RandomDNSName(4)
 			uploadedFiles, err = testenv.UploadFilesToS3(testS3Bucket, s3TestDir, appFileList, downloadDirV1)
 			Expect(err).To(Succeed(), fmt.Sprintf("Unable to upload %s apps to S3 test directory for Standalone", appVersion))
 			uploadedApps = append(uploadedApps, uploadedFiles...)
 
 			// Create App framework spec for Standalone
-			volumeName := "appframework-test-volume-" + testenv.RandomDNSName(3)
-			volumeSpec := []enterpriseApi.VolumeSpec{testenv.GenerateIndexVolumeSpec(volumeName, testenv.GetS3Endpoint(), testenvInstance.GetIndexSecretName(), "aws", "s3")}
-			appSourceDefaultSpec := enterpriseApi.AppSourceDefaultSpec{
-				VolName: volumeName,
-				Scope:   enterpriseApi.ScopeLocal,
-			}
-
-			// Maximum apps to be downloaded in parallel
-			maxConcurrentAppDownloads := 5
-
-			// appSourceSpec: App source name, location and volume name and scope from appSourceDefaultSpec
-			appSourceName := "appframework" + testenv.RandomDNSName(3)
-			appSourceSpec := []enterpriseApi.AppSourceSpec{testenv.GenerateAppSourceSpec(appSourceName, s3TestDir, appSourceDefaultSpec)}
-			appFrameworkSpec := enterpriseApi.AppFrameworkSpec{
-				Defaults:                  appSourceDefaultSpec,
-				AppsRepoPollInterval:      60,
-				VolList:                   volumeSpec,
-				AppSources:                appSourceSpec,
-				MaxConcurrentAppDownloads: uint64(maxConcurrentAppDownloads),
-			}
+			appSourceName = "appframework-" + enterpriseApi.ScopeLocal + testenv.RandomDNSName(3)
```

Comment:
How is `MaxConcurrentAppDownloads` handled in the new API?

Reply:
I will add support for `maxConcurrentAppDownloads` on the feature branch.

Reply:
We are limiting the changes in the PR to backport changes

Reply:
This can achieved by appFrameworkSpec.MaxConcurrentAppDownload := unit64(maxConcurrentAppDownloads) on line 124 and restore the following line.
`// Maximum apps to be downloaded in parallel
maxConcurrentAppDownloads := 5`

Reply:
Changes made